rsrc_packages.Rmd

```diff
@@ -1,20 +1,47 @@
 ## Packages & Docs {#packages}
 
-The [Packages page](https://ropensci.org/packages/) on our website lists our carefully vetted, staff- and community-contributed R tools that lower barriers to working with open data sources for various research applications. Of over 400 packages, about 30.0% have been peer reviewed, and more than half are on CRAN. 
+The [Packages page](https://ropensci.org/packages/) on our website lists our carefully vetted, 
+staff- and community-contributed R tools that lower barriers to working with open data sources for various research applications. 
+Of over 400 packages, about 30.0% have been peer reviewed, and more than half are on CRAN. 
```

Comment:
should this bit come from a code chunk that'd read [the registry JSON file](https://github.com/ropensci/roregistry/blob/gh-pages/registry.json)

Reply:
Yes, good idea! I think I'll open a new issue (#76) and leave if for the next version.